.travis.yml

```diff
@@ -6,4 +6,6 @@ cache:
   directories:
     - "node_modules"
 after_script:
-  - after-build --exit-code $TRAVIS_TEST_RESULT
\ No newline at end of file
+  - after-build --exit-code $TRAVIS_TEST_RESULT
```

Comment:
What changed on this line?
I just want to know why it is tracked

Reply:
I think it's the addition of line 10 and 11

Reply:
nothing changed from my end. I just added `after_success: run coverage` on line 10.